### PR TITLE
Add action to create deb/rpm packages

### PR DESCRIPTION
**Pull request by dctrwatson**
Partially addresses #4955 

Signed-off-by: John Watson <johnw@planetscale.com>

### CODE REVIEW

File: docker/packaging/package_vitess.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+set -eu
+
+inputs_file="/vt/packaging/inputs"
+cat <<EOF >> "${inputs_file}"
+/vt/bin/mysqlctld
+/vt/bin/vtctlclient
+/vt/bin/vtctld
+/vt/bin/vtgate
+/vt/bin/vttablet
+/vt/bin/vtworker
```
Comment (dkhenry) [inline docker/packaging/package_vitess.sh]: Add in /vt/bin/vtbackup

File: docker/packaging/etc_default_vitess [hunk 2]
```diff
@@ -0,0 +1,6 @@
+VTROOT="/vt"
+VTTOP="$VTROOT/src/vitess.io/vitess"
+GOTOP="$VTTOP/go"
+GOBIN="$VTROOT/bin"
+GOPATH="$VTROOT"
```
Comment (dkhenry) [inline docker/packaging/etc_default_vitess]: I don't know if we should have golang variables in the packagesa. I think in a perfect world you don't need to have any go installed to install vitess

File: docker/packaging/package_vitess.sh [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    --version "${VERSION}" \
+    --url "https://vitess.io/" \
+    --description "${description}" \
+    --license "Apache License - Version 2.0, January 2004" \
+    --config-files "/vt/config" \
+    --config-files "/etc/default/vitess" \
+    --directories "/vt/bin" \
+    --directories "/vt/src" \
+    --inputs "${inputs_file}" \
+    --before-install "/vt/packaging/preinstall.sh" \
```
Comment (dkhenry) [inline docker/packaging/package_vitess.sh]: For all of these can we move from Vitess specific directories to the standard unix directories. 
```
config=/etc/vitess
bin=$PREFIX/bin
src=$PREFIX/src
lib=$PREFIX/lib/vitess/ 
```

and have all the web resource go into the lib folder
```

File: docker/packaging/package_vitess.sh (hunk 1 above)
Comment (dkhenry) [reply docker/packaging/package_vitess.sh]: Also  `/vt/bin/vtctl`

File: docker/packaging/package_vitess.sh (hunk 3 above)
Comment (dctrwatson) [reply docker/packaging/package_vitess.sh]: Easy enough to do. I was just mimicking the `lite` docker image.

File: docker/packaging/etc_default_vitess (hunk 2 above)
Comment (dctrwatson) [reply docker/packaging/etc_default_vitess]: It's not for Go itself, but in https://github.com/vitessio/vitess/blob/master/docker/lite/Dockerfile#L38 this comment makes them sound necessary.

`# Set up Vitess environment (just enough to run pre-built Go binaries)`

**Comment by sougou** [comment]
@dkhenry Is this ready to merge?

**Pull request merged**